### DISCUSSION TITLE
customize strapi-admin-content-manager?

### DISCUSSION THREAD

**Discussion by dunods** [strapi/strapi#5919] [Legacy Topics]
how do i customize services strapi-admin-content-manager within node_modules.

i did try to create my-app/extensions/strapi-admin-content-manger/services/ContentTypes.js but it didnt work ?any help

**Comment by derrickmehaffy** [comment]
`my-app/extensions/content-manger/services/ContentTypes.js`

**Comment by dunods** [comment]
![image](https://user-images.githubusercontent.com/8032766/80088683-0b024a00-8587-11ea-8345-ec923fd73b52.png)

already try it shows error 
please assist

**Reply by derrickmehaffy** [reply]
Need more information as to what you are changing

**Reply by dunods** [reply]
i just add more value at const HIDDEN_CONTENT_TYPES 
and added this   'plugins::users-permissions.user'

**Reply by dunods** [reply]
manage to fix it by importing the dependency ... many thanks @derrickmehaffy

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question that is not really a bug report or a feature request, questions should be directed to the [discussions](https://github.com/strapi/strapi/discussions)

I have converted this to a discussion for you, we try to keep Github clean and only for bug reports and feature requests.
